dipy/workflows/viz.py

```diff
@@ -142,9 +145,15 @@ def run(self, input_files, cluster=False, cluster_thr=15.,
                     print('Peak_dirs shape')
                     print(pam.peak_dirs.shape)
 
+        if len(bg_color) == 1:
```

Comment:
This will work only if bg_color is a tuple (0.5,) are you really supporting this option? Not explained in the docstring above.

Reply:
Good catch! I will explain it in the docstring. This option is for lazy people. Instead of doing `--bg_color 0.5 0.5 0.5` you can do `--bg_color 0.5`

Reply:
it will work, because `variable float` return always a list